forms-flow-nav/src/Navbar.jsx

```diff
@@ -137,6 +137,21 @@ const NavBar = React.memo(({ props }) => {
   const [loginUrl, setLoginUrl] = useState(baseUrl);
 
   const logoPath = MULTITENANCY_ENABLED ? tenantLogo : defaultLogoPath;
+  
+  useEffect(() => {
+    const data = JSON.parse(StorageService.get("TENANT_DATA"));
+    if (MULTITENANCY_ENABLED && data?.details) {
+      setTenantLogo(data?.details?.customLogo?.logo || "/logo.svg");
+      let link = document.querySelector("link[rel~='icon']");
+      if (!link) {
+        link = document.createElement("link");
+        link.rel = "icon";
+        document.getElementsByTagName("head")[0].appendChild(link);
+      }
+      link.href = tenantLogo;
+    }
+  }, [tenant]);
```

Comment:
give the data also in dependencies array

Reply:
added